src/Splitter.sol

```diff
@@ -39,7 +39,7 @@ contract Splitter {
     mapping (address => uint256) public wards;
     uint256     public           live;    // Active Flag
     FlapLike    public           flapper; // Underlying burner strategy
-    FarmLike    public           farm;    // StakingRewards contract receiving DAI rewards
+    FarmLike    public           farm;    // StakingRewards contract receiving the rewards
```

Comment:
Not opposed to do the changed, but is there anything wrong with "DAI"?

Reply:
Capitalized DAI is assumed as the dai token.

Reply:
Hmm yeah the whole thing is based on dai/DAI today, so this will need to be migrated, but yeah better to have one less place to change it.

Reply:
The migration should only be searching for `nst`, `Nst` and `NST`.
Something like `daiJoin` in the contract should be a generalized name for DAI's join and NST's join.